src/Validator/Rules/QueryComplexity.php

```diff
@@ -206,11 +206,16 @@ static function ($error) {
 
                 return ! $directiveArgsIf;
             }
-            $directive       = Directive::skipDirective();
-            $directiveArgsIf = Values::getArgumentValues($directive, $directiveNode, $variableValues);
+            if ($directiveNode->name->value === 'skip') {
```

Comment:
can use constant here
```suggestion
            if (Directive::SKIP_NAME === $directiveNode->name->value) {
```